### PR TITLE
[Documentation][Book] Address Book

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no
| New docs?    | yes
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/address_book.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. index::
+   single: Address Book
+
+Address Book
+============
+
+The Address Book concept is a very convenient solution for the customers of your shop, that come back.
```
Comment (NeverResponse) [inline docs/book/address_book.rst]: `... for the users that frequent your shop.`

File: docs/book/address_book.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Once they provide an address it is saved in the system and can be reused the next time.
+
+**Sylius** handles the address book in a very simple way. We are holding a collection of addresses on the Customer entity.
+
+The Addresses Collection on a Customer
+--------------------------------------
+
+On the Customer entity we are holding a collection of addresses:
```
Comment (NeverResponse) [inline docs/book/address_book.rst]: Think the title got this one covered

File: docs/book/address_book.rst (hunk 1 above)
Comment (lchrusciel) [inline docs/book/address_book.rst]: `for your loyal customers`?

File: docs/book/address_book.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.. code-block:: php
+
+    /**
+     * @var Collection|AddressInterface[]
+     */
+    protected $addresses;
+
+We can operate on this collection as usually - by adding and removing objects.
+
+Besides the Customer entity has a **default address** field that is the default address used both for shipping and billing,
```
Comment (lchrusciel) [inline docs/book/address_book.rst]: There is no such a filed like `default address`. Customer entity has two fields which with default addresses and they are `billingAddress` and `shippingAddress`

File: docs/book/address_book.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+the one that will be filling the form fields by default.
+
+How to add an address to the address book manually?
```
Comment (lchrusciel) [inline docs/book/address_book.rst]: Usually, we have a whole example with factories here.

File: docs/book/address_book.rst (hunk 3 above)
Comment (TheMadeleine) [inline docs/book/address_book.rst]: I suppose that in the current rework we are adding this approach.

**Comment by pjedrzejewski** [comment]
Thank you Magda, great work! :)

**Pull request merged**